src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -1076,14 +1080,26 @@ def dbapi(
         max_workers: Optional[int] = None,
         query_timeout: Optional[int] = 0,
         fetch_size: Optional[int] = 0,
+        custom_schema: Optional[Union[str, StructType]] = None,
```

Comment:
in this design, this parameter have to be exact same column names as that in sql server/oracle db,
Do you think we should allow user to input a map like: {data_source_column_name_string -> StructType("new name", data_type)}, so that user can rename during read? They could also possibly only pick a part of columns(we can provide this functionality later)

Reply:
this is alignment with pyspark. what you describe could be a new feature